### PR TITLE
Don't install service files instead mention to user how to do this themselves

### PR DESCRIPTION
**Pull request by vmcj**
This now implements what was mentioned in the linked issue, this was updated from before.

Closes: https://github.com/DOMjudge/domjudge/issues/2086

### CODE REVIEW

File: doc/manual/install-judgehost.rst [hunk 1]
```diff
@@ -231,3 +231,6 @@ the judgedaemon.
 The judgedaemon can also be run as a service by running::
 
   sudo systemctl enable --now domjudge-judgehost
+
+This only works in a FHS install, if the judgehost has been installed with another
+option those `service` files have to manually copied to the right directory e.g. `/etc/systemd`.
```
Comment (eldering) [inline doc/manual/install-judgehost.rst:236]: Improve punctuation:
```suggestion
option, those `service` files have to manually copied to the right directory,
e.g. `/etc/systemd`.
```

**Comment by thijskh** [comment]
I’m all for making docs match reality, but seems to complicatie the instructions and increase mental load. What is the relation with [this idea](https://github.com/DOMjudge/domjudge/issues/2086#issuecomment-1826742165) to make the software less complex, instead of the instructions more complex?

**Comment by vmcj** [comment]
> I’m all for making docs match reality, but seems to complicatie the instructions and increase mental load. What is the relation with [this idea](https://github.com/DOMjudge/domjudge/issues/2086#issuecomment-1826742165) to make the software less complex, instead of the instructions more complex?

So you want to leave the documentation as is and mention it after the `make install-judgehost` step in code?

**Comment by eldering** [comment]
> Yes, as I understand that this is Jaap’s suggestion in the linked issue

And also remove the installation of service files for the FHS install.

File: doc/manual/install-judgehost.rst (hunk 1 above)
Comment (vmcj) [reply doc/manual/install-judgehost.rst]: Fixed by not mentioning this at all.

File: Makefile [hunk 2]
```diff
@@ -167,6 +167,17 @@ endif
...
 		etc/restapi.secret
+	@echo ""
+	@echo "========== Judgehost Install Completed =========="
+	@echo ""
+	@echo "Optionally:"
+	@echo "    - Install the create-cgroup service to setup the secure judging restrictions:"
+	@echo "        cp <DOMSERVER_UNPACK_PATH>/judge/create-cgroups.service /etc/systemd/system/"
+	@echo "    - Install the judgehost service:"
+	@echo "        cp <DOMSERVER_UNPACK_PATH>/judge/domjudge-judgedaemon@.service /etc/systemd/system/"
+	@echo "    - You can enable the judghost on core 1 as:"
```
Comment (eldering) [inline Makefile:178]:
```suggestion
	@echo "    - You can enable the judgehost on CPU core 1 with:"
```

File: doc/manual/install-judgehost.rst [hunk 3]
```diff
@@ -68,6 +68,16 @@ example to install DOMjudge in the directory ``domjudge`` under `/opt`::
   make judgehost
   sudo make install-judgehost
 
+Example service files for the judgehost and the judgedaemon are provided in
+``judge/create-cgroups.service`` and ``judge/domjudge-judgedaemon@.service``. The rest of the manual assumes you install those
+in a location which is picked up by ``systemd`` for example ``/etc/systemd/system``.
```
Comment (eldering) [inline doc/manual/install-judgehost.rst:73]:
```suggestion
in a location which is picked up by ``systemd``, for example ``/etc/systemd/system``.
```

File: Makefile [hunk 4]
```diff
@@ -167,6 +167,17 @@ endif
 # Fix permissions and ownership for password files:
 	-$(INSTALL_USER) -m 0600 -t $(DESTDIR)$(judgehost_etcdir) \
 		etc/restapi.secret
+	@echo ""
+	@echo "========== Judgehost Install Completed =========="
+	@echo ""
+	@echo "Optionally:"
+	@echo "    - Install the create-cgroup service to setup the secure judging restrictions:"
+	@echo "        cp <DOMSERVER_UNPACK_PATH>/judge/create-cgroups.service /etc/systemd/system/"
```
Comment (eldering) [inline Makefile:175]: In other places we don't refer to the base path of the installation. Also this is normally run from that directory, so I'd remove the `<DOMSERVER_UNPACK_PATH>/`, also in other places in this PR.

**Review by eldering — APPROVED** [review]

**Pull request merged**